File: app/src/main/java/com/example/globoplay_desafio_mobile/response/movies/MovieResponse.java
package com.example.globoplay_desafio_mobile.response.movies;

import com.example.globoplay_desafio_mobile.models.movies.MovieModel;
import com.google.gson.annotations.SerializedName;
import java.util.ArrayList;

public class MovieResponse {

    @SerializedName("results")
    private ArrayList<MovieModel> movies;
    private int total_pages;

    // Getter
    public ArrayList<MovieModel> getMovies() {
        return movies;
    }

    public int getTotal_pages() {
        return total_pages;
    }
}

File: app/src/main/java/com/maslima/globo_play_recrutamento/network/responses/MovieResponse.kt
package com.maslima.globo_play_recrutamento.network.responses

import com.google.gson.annotations.SerializedName
import com.maslima.globo_play_recrutamento.network.model.MovieDto

data class MovieResponse(
    @SerializedName("page") val page : Int,
    @SerializedName("results") val movies : List<MovieDto>,
    @SerializedName("total_pages") val total_pages : Int,
    @SerializedName("total_results") val total_results : Int
)